src/javascript/app/base/page.js

```diff
@@ -86,6 +87,7 @@ const Page = (() => {
             updateLinksURL('body');
             recordAffiliateExposure();
             endpointNotification();
+            InterviewPopup.onLoad();
```

Comment:
I think we also don't want to call this on isLoginPages() similar to line 81, so we should move it or if we want to keep the hierarchy of function calls we should store isLoginPages() it in a variable and check that again here

Reply:
i'll move it to `!Login.isLoginPages` if bracket